Mathlib/Data/Nat/Factorial/Basic.lean

```diff
@@ -207,6 +207,14 @@ theorem factorial_mul_pow_sub_le_factorial {n m : ℕ} (hnm : n ≤ m) : n ! * n
     _ ≤ _ := by simpa [hnm] using @Nat.factorial_mul_pow_le_factorial n (m - n)
 #align nat.factorial_mul_pow_sub_le_factorial Nat.factorial_mul_pow_sub_le_factorial
 
+lemma factorial_le_pow : ∀ n, n ! ≤ n ^ n
```

Comment:
Why are you sneaking in this unrelated lemma?

Reply:
because it's not completely unrelated and I'm a bit bored of opening hundreds of tiny PRs. I can split it off if you really insist.

Reply:
It's okay now that you mentioned it in PR commit message.